<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QDialog" name="FilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>187</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>187</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>492</width>
    <height>187</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filters</string>
  </property>
  <widget class="QDialogButtonBox" name="FilterForm_ButtonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>471</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="FilterForm_PlayStatusGrp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>143</width>
     <height>134</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>134</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>143</width>
     <height>134</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Play Status</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="FilterForm_NotPlayed">
      <property name="minimumSize">
       <size>
        <width>119</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Show Not Played</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="FilterForm_Played">
      <property name="minimumSize">
       <size>
        <width>119</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Show Played</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="FilterForm_Complete">
      <property name="minimumSize">
       <size>
        <width>119</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Show Complete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="FilterForm_Abandoned">
      <property name="minimumSize">
       <size>
        <width>119</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Show Abandoned</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="FilterForm_PlatformGrp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>131</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>131</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Platform</string>
   </property>
   <widget class="QCheckBox" name="FilterForm_PC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show PC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FilterForm_PS4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Playstation 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FilterForm_PS3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>78</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Playstation 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FilterForm_PSVita">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>104</y>
      <width>146</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Playstation Vita</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FilterForm_OwnedGrp">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>139</width>
     <height>134</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>139</width>
     <height>134</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>139</width>
     <height>134</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Owned Status</string>
   </property>
   <widget class="QCheckBox" name="FilterForm_Owned">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Owned</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FilterForm_NotOwned">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>119</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Not Owned</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>FilterForm_ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FilterForm_ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
